lib/controller.php

```diff
@@ -185,6 +185,14 @@ public function import_blob($blob) {
 			'_sha'         => $blob->sha,
 		) );
 
+		if(class_exists("WPCom_Markdown")) {
+			$wpcomMd = WPCom_Markdown::get_instance();
+			$args = $wpcomMd -> wp_insert_post_data($args, array(
+				"ID"           => $this->id,
+				"post_type"    => $args['post_type']
```

Comment:
Ugh.. That is a problem. I need the type or the plugin won't be able to do things correctly
Can I use the `layout` attribute instead?